dnf-plugins/py3query.py

```diff
@@ -268,7 +269,14 @@ def bug_namegetter(bug):
                 status = bug.status
                 if bug.resolution:
                     status += ' ' + bug.resolution
-                r.setdefault('links', {})['bug'] = [bug.weburl, status]
+                # Let's get the datetime of the last comment
+                last_comment_datetime = None
+                if bug.comments:
```

Comment:
That would mean that if someone changes some little detail—for example if we change the Blocks field it—looks like it's active. I'll try to measure how much slower this makes the execution, because I don't remember it being that significant, but maybe it was.